circuit_tutorials/plonky2/merkle_tree/README.md

```diff
@@ -0,0 +1,8 @@
+# Merkle Tree Circuit
+
+This Rust script uploads and proves a Plonky2 circuit that verifies a Merkle inclusion proof for a 1024 leaf Merkle tree
```

Comment:
Sentence missing final punctuation
s/1024 leaf/1024-leaf/
Is there anything else we can elaborate on in this readme about the script design, input format, or such? Perhaps take a cue from the existing tutorial README.md contents?

Reply:
Done. [fe85d96](https://github.com/Sindri-Labs/sindri-resources/pull/98/commits/fe85d96484a449d80ebb69276067212dd4267c12)

Reply:
Much better, thanks.